index.js

```diff
@@ -98,13 +98,20 @@ module.exports = {
       portRequest = 0;
     }
 
-    await server.start({ port: portRequest });
+    let hostnameRequest;
```

Comment:
It should be possible to remove this. It looks like the server falls back to localhost if the hostname property is falsy. In any case, I'd rather have only one copy of the hostname fallback logic.